niviz/interfaces/views.py

```diff
@@ -48,3 +50,75 @@ def _post_run_hook(self, runtime):
 
     def _run_interface(self, runtime):
         return runtime
+
+
+class _ISegInputSpecRPT(nrc._SVGReportCapableInputSpec,
+                        BaseInterfaceInputSpec):
+    '''
+    Input specification for ISegRPT, implements:
+
+    anat_file: Input anatomical image
+    seg_files: Input segmentation image(s) - can be a list or a single file
+    mask_file: Input ROI mask
+
+    Bases _SVGReportCapableInputSpec which implements:
+
+    out_report: Filename trait
+    compress_report: ["auto", true, false]
+
+    '''
+    anat_file = File(exists=True,
+                     usedefault=False,
+                     resolve=True,
+                     desc='Anatomical image of SVG',
+                     mandatory=True)
+
+    seg_files = InputMultiPath(
+                    File(exists=True,
+                         usedefault=False,
+                         resolve=True),
+                    desc='Segmentation image of SVG',
+                    mandatory=True)
+
+    mask_file = File(exists=True,
+                     udedefault=False,
+                     resolve=True,
+                     desc='ROI Mask for mosaic',
+                     mandatory=True)
```

Comment:
there's a workaround for this, see the other comment at the end

```suggestion
    mask_file = File(exists=True,
                     resolve=True,
                     desc='ROI Mask for mosaic')
```